src/knex.ts

```diff
@@ -0,0 +1,7 @@
+import { knex as Knex } from 'knex'
+import env from './env'
+
+export default Knex({
+	client: 'pg',
+	connection: env.DATABASE_URI
+})
```

Comment:
Won’t this error if `connection` is `''`?

Reply:
I just tested it and it doesn't cause problem as long as I don't import it anywhere, need to fix...